ruby.c

```diff
@@ -334,11 +334,13 @@ usage(const char *name, int help, int highlight, int columns)
     };
 #if USE_YJIT
     static const struct ruby_opt_message yjit_options[] = {
-        M("--yjit-stats",              "", "Enable collecting YJIT statistics"),
-        M("--yjit-exec-mem-size=num",  "", "Size of executable memory block in MiB (default: 64)"),
-        M("--yjit-call-threshold=num", "", "Number of calls to trigger JIT (default: 30)"),
-        M("--yjit-max-versions=num",   "", "Maximum number of versions per basic block (default: 4)"),
-        M("--yjit-greedy-versioning",  "", "Greedy versioning mode (default: disabled)"),
+        M("--yjit-stats",                    "", "Enable collecting YJIT statistics"),
+        M("--yjit-trace-exits",              "", "Trace side exit locations"),
+        M("--yjit-trace-exits-sample-rate",  "", "Trace side exit locations sampling at the rate specified"),
```

Comment:
```suggestion
        M("--yjit-trace-exits-sample-rate",  "", "Trace exit locations only every Nth occurrence"),
```

1. Side exits are not equivalent to exits. This trace all exits. The "side" part is more about the placement of the exit code.
2. This might just be me but it's not immediately clear to me what the units for "rate" is. There might be better wording than what I put.